lazycatlabs/auth_api
# fix: show error message when bad request

## What it does

- fix merge-ci GitHub action
- show the error message when the bad request
- improve validator use required
- require a general token to send email

## How to test

1. remove some params and send the request
2. observe the response
3. hit send email API without general token
4. observe the response

## Screenshot

![image](https://github.com/user-attachments/assets/a77b133f-164f-4fcc-9a2e-762a07232a1f)


![image](https://github.com/user-attachments/assets/761ea414-7bd6-4133-b9af-2f9b3f39322f)
